network/suite.go

```diff
@@ -63,3 +83,94 @@ func (s *TestSuite) CreateTxBytes(fees sdk.Coin, gas uint64, msgs []sdk.Msg) ([]
 	bz, err := val.ClientCtx.TxConfig.TxEncoder()(builder.GetTx())
 	return bz, err
 }
+
+func (s *TestSuite) GetAccountI(acc account.Account) (sdk.AccountI, error) {
+	cc, closeFn, err := s.GetGRPC()
+	if err != nil {
+		return nil, err
+	}
+	defer closeFn()
+
+	authClient := authtypes.NewQueryClient(cc)
+
+	resp, err := authClient.Account(context.Background(), &authtypes.QueryAccountRequest{Address: acc.Address().String()})
+	if err != nil {
+		return nil, err
+	}
+
+	var accI sdk.AccountI
+	if err := authCodec.UnpackAny(resp.Account, &accI); err != nil {
+		return nil, err
+	}
+
+	return accI, err
+}
+
+func (s *TestSuite) GetCometClient() (*cmthttp.HTTP, error) {
+	return cmthttp.New(s.Network.Validators[0].RPCAddress, "/websocket")
+}
```

Comment:
Is there any case where we would want to get the rpc address of a validator other than the first validator?

Reply:
Not sure actually. @Eric-Warehime @nivasan1 thoughts?

Reply:
I guess as an idea...if we wanted to fill the mempool of all validators at once we'd need each thread w/ a different RPC connection sending txs to that validator.

Not sure what our plans are for running larger scale tests w/ chaintestutil or if we're just targeting 1 node networks for now?

Reply:
IDK, in the event that we do, it's exported, so the consumers can still do that